Fix building wxGTK/Win32

stahta01 opened the pull request:
Fix building wxGTK/Win32 for MSys2

Commit:
Update wxGTK/Win32 configure/make directions

Commit:
Add GDK_WINDOWING_X11 guard for wxGTK/Win32

PBfordev reviewed (commented):
Doesn't this clash with the rewrite of build instructions in #960, particularly with https://github.com/wxWidgets/wxWidgets/pull/960/files#diff-e71daf45b72a83b5598ab8196ab17df1

stahta01 commented:
Yes, it would clash with that PR because I rewrote my directions to make them safer; and, added GTK3

I will remove the directions change if that is requested by an wx development team member.

Tim S.

stahta01 commented:
Looks like my fix has a problem; so, I will close the PR.

Tim S.
